servers/audio/effects/audio_effect_reverb.cpp

```diff
@@ -29,6 +29,7 @@
 /**************************************************************************/
 
 #include "audio_effect_reverb.h"
+// #include "audio_effect_reverb.compat.inc"
```

Comment:
I don't believe this comment is necessary

Reply:
Oh right forgot these. Sorry